doc/sphinxdoc/models.rst

```diff
@@ -0,0 +1,653 @@
+.. Essentia TensorFlow models
+
+Essentia TensorFlow models
+==========================
+
+This is a list of pre-trained TensorFlow models available in Essentia for various (music) audio analysis and classification tasks. To use the models in Essentia, see the `machine learning inference <machine_learning.html>`_ page.
+
+
+All the models created by the MTG are licensed under CC BY-NC-SA 4.0 (https://creativecommons.org/licenses/by-nc-sa/4.0/) and are also available under proprietary license upon request (https://www.upf.edu/web/mtg/contact).
+
+
+The models are serialized in `Protocol Buffer <https://developers.google.com/protocol-buffers/>`_ files suitable for inference with Essentia and TensorFlow. Additionally, some models are also available in `TensorFlow.js <https://www.tensorflow.org/js/models>`_) format. Each ``.pb`` file is coupled with a ``.json`` file containing its metadata.
+
+
+Model file naming conventions:
+
+- classifiers: ``<target_task>-<architecture>-<source_task>-<version>.pb``
+- MusiCNN: ``<task>-<architecture>-<version>.pb``
+- OpenL3: ``<architecture>-<params>-<version>.pb``
+- Spleeter: ``<architecture>-<params>-<version>.pb``
+- TempoCNN: ``<architecture>-<params>-<version>.pb``
+- VGG: ``<task>-<architecture>-<version>.pb``
+- VGGish: ``<task>-<architecture>-<version>.pb``
+- YAMNet: ``<task>-<architecture>-<version>.pb``
+
+Where:
+
+- ``<architecture>``: the architecture of the model (refers only to the source model on the transfer learning classifiers).
+- ``<task>`` and ``<source-task>``: the dataset in which the model, or the source model was trained.
+- ``<target-task>``: the dataset in which the target model was trained (only for the transfer learning classifiers).
+- ``params``: parameters specific to a particular family of models.
+- ``version``: represents the incremental version of the model. We do not guaranty full compatibility between different versions.
+
+
+
+Audio event recognition
+^^^^^^^^^^^^^^^^^^^^^^^
+Download: https://essentia.upf.edu/models/audio-event-recognition/
```

Comment:
Minor thing: I don't like much how this looks.
[Download](https://essentia.upf.edu/models/audio-event-recognition/)
or..

[See all the models](https://essentia.upf.edu/models/audio-event-recognition/)

Reply:
"Download" is probably too short to be easily noticed. "See all the models" would suggest there is more info about the models than on this page.

What about "Download model files" or "Download models"?

Reply:
I like Download model files